src/libaktualizr/uptane/imagesrepository.h

```diff
@@ -29,9 +29,13 @@ class ImagesRepository : public RepositoryCommon {
 
   Exception getLastException() const { return last_exception; }
 
+  static std::shared_ptr<Uptane::Targets> verifyDelegation(const std::string& delegation_raw, const Uptane::Role& role,
+                                                           const Targets& parent_target);
+  std::shared_ptr<Uptane::Targets> getTrustedTargets() { return targets; }
```

Comment:
"Trusted targets" is an interesting name. Based on the line `auto toplevel_targets = images_repo.getTrustedTargets();` it appears to be just the toplevel targets; is that accurate? Maybe a comment in the header to explain what is meant by "trusted" here would be helpful.